R/summary-log.R

```diff
@@ -212,15 +271,16 @@ extract_heuristics <- function(.s) {
 #' @describeIn extract_from_summary Extract objective function value (without constant) for the final estimation method
 #' @importFrom purrr map map_dbl
 #' @keywords internal
-extract_ofv <- function(.s) {
+extract_ofv <- function(.s, with_constant = FALSE) {
+  name <- if (with_constant) "ofv_with_constant" else "ofv_no_constant"
```

Comment:
missing `@param` documentation for `with_constant`. May want to update the title too since it says "(without constant)"

Reply:
Thanks for spotting, updated.